rq/cli/cli.py

```diff
@@ -227,6 +227,11 @@ def worker(
     sentry_debug = sentry_debug or settings.get('SENTRY_DEBUG')
     sentry_dsn = sentry_dsn or settings.get('SENTRY_DSN')
     name = name or settings.get('NAME')
+    dict_config = options.get("dict_config", settings.get('DICT_CONFIG'))
+    
+    if dict_config:
+        import logging.config
```

Comment:
This import can be moved to module level.

Reply:
Done